<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1235</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tasksTab">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Задачи</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QDateEdit" name="taskDateEdit">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="employeeBox">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTableWidget" name="taskTable">
            <column>
             <property name="text">
              <string>Нар./Расп.</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Виды работ</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Блок</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Помещения</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Оборудование</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Производитель</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QToolButton" name="toolButton">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_2">
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Работы</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="rasp_tab">
       <attribute name="title">
        <string>Распоряжения</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QDateEdit" name="raspDateEdit">
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QTableWidget" name="raspTable">
            <column>
             <property name="text">
              <string>№</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Производитель</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Члены бригады</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Блок</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Оборудование</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QToolButton" name="addRaspButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/icons/icons/add.png</normaloff>:/icons/icons/add.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="editRaspButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="deleteRaspButton">
              <property name="text">
               <string>...</string>
              </property>
              <property name="icon">
               <iconset resource="resource.qrc">
                <normaloff>:/icons/icons/delete.png</normaloff>:/icons/icons/delete.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>32</width>
                <height>32</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1235</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Подключиться к базе</string>
     </property>
     <addaction name="aDefault"/>
     <addaction name="aAnother"/>
    </widget>
    <addaction name="menu_3"/>
    <addaction name="aError"/>
    <addaction name="aExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="aEmployees"/>
    <addaction name="aUnits"/>
    <addaction name="aSchedule"/>
    <addaction name="aLocations"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Документы</string>
    </property>
    <addaction name="aDocuments"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="aAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="aEmployees">
   <property name="text">
    <string>Работники...</string>
   </property>
  </action>
  <action name="aUnits">
   <property name="text">
    <string>Блоки...</string>
   </property>
  </action>
  <action name="aDocuments">
   <property name="text">
    <string>Сформировать пакет документов</string>
   </property>
  </action>
  <action name="aDefault">
   <property name="text">
    <string>По умолчанию</string>
   </property>
  </action>
  <action name="aAnother">
   <property name="text">
    <string>Другая БД...</string>
   </property>
  </action>
  <action name="aExit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="aAbout">
   <property name="text">
    <string>О программе...</string>
   </property>
  </action>
  <action name="aError">
   <property name="text">
    <string>Журнал ошибок...</string>
   </property>
  </action>
  <action name="aSchedule">
   <property name="text">
    <string>График работ...</string>
   </property>
  </action>
  <action name="aLocations">
   <property name="text">
    <string>Расположение оборудования...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>aExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>617</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
